platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/item/message/WorkflowItemMessageRow.java

```diff
@@ -94,7 +94,6 @@ public int compareTo(WorkflowItemMessageRow other) {
 
 		return Comparator//
 			.comparing(WorkflowItemMessageRow::getTransaction)
-			.thenComparing(WorkflowItemMessageRow::getText)
```

Comment:
This change implies the assumption that there will always be at most one `WorkflowItemMessageRow` for any given `AGTransaction`. I wonder if that assumption is always fulfilled.

Reply:
No, that is not the point, there can be more than one `WorkflowItemMessageRow` per `AGTransaction`. This is why the loading order is important. `WorkflowItemMessageRow` records with the same `AGTransaction` need to be order like this:
- comments and tasks
- auto transition source states
- item logs